tests/test_prompts.py

```diff
@@ -1,84 +1,80 @@
-        *DEFAULT_PROMPTS,
-        {
-            "label": "Custom Prompt",
-            "description": "This is a custom prompt",
-            "prompt": "The prompt",
-            "method": "append",
-        },
-    ]
 
-    assert get_prompts()[-1].label == "Custom Prompt"
+def test_editing_default_prompts():
+    prompts = get_prompts()
+    assert prompts[0]["label"] == DEFAULT_PROMPTS[0]["label"]
+    assert prompts[0]["prompt"] == DEFAULT_PROMPTS[0]["prompt"]
 
+    default_prompt = Prompt.objects.get(
+        default_prompt_id=DEFAULT_PROMPTS[0]["default_prompt_id"]
+    )
 
-def test_invalid_prompt(settings):
-    settings.WAGTAIL_AI_PROMPTS = [
-        {
-            "description": "This is a custom prompt",
-            "prompt": "The prompt",
-        }
-    ]
+    default_prompt.prompt = "New Prompt Text"
+    default_prompt.save()
+    assert default_prompt.prompt_value == "New Prompt Text"
 
-    with pytest.raises(ImproperlyConfigured):
-        assert get_prompts()
+    prompts = get_prompts()
+    assert prompts[0]["prompt"] == "New Prompt Text"
+    assert prompts[0]["prompt"] != DEFAULT_PROMPTS[0]["prompt"]
 
 
-def test_invalid_setting_structure(settings):
-    settings.WAGTAIL_AI_PROMPTS = {
-        "label": "Custom Prompt",
-        "description": "This is a custom prompt",
-        "prompt": "The prompt",
-        "method": "append",
-    }
+def test_get_prompts_returns_new_prompts_and_default_prompts(setup_prompt_object):
+    prompts = get_prompts()
+    assert (
+        len(prompts) == len(DEFAULT_PROMPTS) + 1
+    )  # 1 Prompt from setup_prompt_object is added
 
-    with pytest.raises(ImproperlyConfigured):
-        assert get_prompts()
+    # Check setup_prompt_object is in get_prompts
+    assert setup_prompt_object.label in [prompt["label"] for prompt in prompts]
+    assert setup_prompt_object.prompt in [prompt["prompt"] for prompt in prompts]
 
+    # Check DEFAULT_PROMPTS are in get_prompts
+    for prompt in DEFAULT_PROMPTS:
+        assert prompt["label"] in [prompt["label"] for prompt in prompts]
+        assert prompt["prompt"] in [prompt["prompt"] for prompt in prompts]
 
-def test_invalid_method(settings):
-    settings.WAGTAIL_AI_PROMPTS = {
-        "label": "Custom Prompt",
-        "description": "This is a custom prompt",
-        "prompt": "The prompt",
-        "method": "foo",
-    }
 
-    with pytest.raises(ImproperlyConfigured):
-        assert get_prompts()
+def test_prompts_return_uuids_and_not_ids():
+    prompts = get_prompts()
+    assert prompts[0]["uuid"] is not None
+    assert "id" not in prompts[0]
+
+
+def test_prompts_can_not_save_and_invalid_method(test_prompt_values):
+    with pytest.raises(ValidationError):
+        prompt = Prompt.objects.create(
+            label=test_prompt_values["label"],
+            description=test_prompt_values["description"],
+            prompt=test_prompt_values["prompt"],
+            method="foo",
+        )
+        prompt.full_clean()
```

Comment:
It's a weird flow to use `create()` and then `full_clean()`, but I guess it does tests the `full_clean()` method.

Instead could setup a model instance without saving it and then call clean().